app.config.ts

```diff
@@ -0,0 +1,18 @@
+// Typing for @expo/config is currently experimental(https://docs.expo.io/workflow/configuration/#using-typescript-for-configuration-appconfigts-instead-of)
+// Install when on stable release (https://www.npmjs.com/package/@expo/config)
+export default ({ config }: any): any => {
+  return {
+    ...config,
+    version: process.env.APP_BINARY_VERSION,
+    android: {
+      ...config.android,
+      versionCode: process.env.APP_BUILD_VERSION
+        ? parseInt(process.env.APP_BUILD_VERSION)
+        : 38
+    },
+    ios: {
+      ...config.ios,
+      buildNumber: process.env.APP_BUILD_VERSION
```

Comment:
what will this fallback to?

Reply:
err in all the cd cases, there will never be any undefined for those env varibles. Might be a problem if someone is trying to manually deploy. Hmm we can possiblely add a default value! What do you suggest?

Reply:
As you mentioned about the error, can we perform the check and throw appropriate errors? So when devs run it locally, they will know why it's failing.